src/Resume/Resume.jsx

```diff
@@ -0,0 +1,185 @@
+import './Resume.css';
+import rating_4 from '../images/rating_4.png';
+import rating_5 from '../images/rating_5.png';
+import right_arrow from '../images/right_arrow.png';
+import { useEffect, useState } from "react";
+import { useLocation, useNavigate } from 'react-router-dom';
+import { useProxy } from "../hooks/useProxy";
+import foto from "../images/foto.jpg"
+import Topic from './Topic';
+
+
+
+function Resume() {
+
+  const [info, setInfo]= useState({});
+  const location = useLocation();
+  const proxy = useProxy();
+  const navigate = useNavigate();
+
+  useEffect(()=>{
+    console.log(location.pathname);
+    const personalInfo = proxy.renderPage(location.pathname);
+    setInfo(personalInfo)
+  }, [location.pathname]);
+
+  function goToEnglishHandler(){
+    navigate("/")
+  }
+
+  function goToGermanHandler(){
+    navigate("/german")
+  } 
+
+  return (
+    <div className='container'>
+
+        <div className="personal_data">
+          
+          <p className="name">{info.name}</p>
+          <div >
+            <img className='photo_img' alt="" src={foto} />
+          </div>
+          <div>
+
+          </div>
+         
+          <div className="info">
+              <p className="heading">{info.personalTitel}</p>
+            
+              <p className='house'>
+              <span className='after_icon'>{info.address}</span></p>
+              <p className='mobile'>
+                <span className='after_icon'>{info.mobile}</span></p>
+              <p className='email'>
+                <span className='after_icon'>{info.email}</span>
+                </p>
+          </div>
+          <hr className='line_left'></hr>
+
+          <div className="info">
+              <p className="heading">{info.progLanguages}</p>
+              <div>{info.html}</div>
+              <div>{info.css}</div>
+              <div>{info.js} </div>
+              <div>{info.react}</div>
+          </div>
+          <hr className='line_left'></hr>
+
+          <div className="info">
+              <p className="heading">{info.personalSkills}</p>
+              <div>{info.skillsCommunication}</div>
+              <div>{info.skillsAnalytical}</div>
+              <div>{info.skillsResponsibility}</div>
+              <div>{info.skillsPurpose}</div>
+          </div>
+          <hr className='line_left'></hr>
+
+          <div className="info">
+              <p className="heading">{info.languages}</p>
+              <div className='rating_home'>  
```

Comment:
Das kann man auch in eine eingene Komponente reinpacken.

```
<div className='rating_home'>  
      <div className='eng give_width'>
      <span className='after_icon'>{info.langEng}</span></div>
      <div> 
        <img className='rating_star' src={rating_4} alt="4 stars" />
      </div>
</div>
```

Reply:
Habe gemacht. Languages component wurde geschrieben.